### PR TITLE
Matt is love

### PR DESCRIPTION
**Pull request by mananchawla2005**
Merge

### CODE REVIEW

**Review by nhcarrigan — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,14 +1,10 @@
-# Project Name
+## We love Matt
```
Comment (TheBusyBiscuit) [inline README.md:1]: Should probably be "Reasons why we love `@MattIPv4`" in my opinion.
And then a link to his profile for his name.

File: README.md [hunk 2]
```diff
@@ -1,14 +1,10 @@
-# Project Name
+## We love Matt 
 
-Project Description
+### Multitalented Person ❤️
+### Contributed in Rythym Project ❤️❤️
+### Hacktoberfest ❤️❤️❤️
+### Digital Ocean ❤️❤️❤️❤️
+### cdnjs ❤️❤️❤️❤️❤️
```
Comment (TheBusyBiscuit) [inline README.md:7]: Those were probably meant to be bullet points, not headings.

File: README.md [hunk 3]
```diff
@@ -1,14 +1,10 @@
-# Project Name
+## We love Matt 
 
-Project Description
+### Multitalented Person ❤️
+### Contributed in Rythym Project ❤️❤️
```
Comment (TheBusyBiscuit) [inline README.md:4]: *Rhythm

File: README.md (hunk 2 above)
Comment (TheBusyBiscuit) [inline README.md:7]: Links to these projects would be cool too

File: README.md [hunk 4]
```diff
@@ -1,14 +1,10 @@
...
 
-Project Description
+### Multitalented Person ❤️
+### Contributed in Rythym Project ❤️❤️
+### Hacktoberfest ❤️❤️❤️
+### Digital Ocean ❤️❤️❤️❤️
+### cdnjs ❤️❤️❤️❤️❤️
 
-## Live Version
+👢🅾️♈️📧 💹🅾️⛎ ♏️🅰️✝️
```
Comment (TheBusyBiscuit) [inline README.md:9]: what.

**Review by TheBusyBiscuit — COMMENTED** [review]
I know this has already been merged, but there are some things I'd like to address.
Especially the typo

File: README.md (hunk 4 above)
Comment (nhcarrigan) [reply README.md:9]: what.

File: README.md (hunk 2 above)
Comment (nhcarrigan) [reply README.md:7]: Happen to have a link to the Rhythm one?

Comment (TheBusyBiscuit) [reply README.md:7]: https://rythmbot.co/

**Comment by TheBusyBiscuit** [comment]
Also *DigitalOcean, not Digital Ocean.
Matt literally opened a PR in another repository fixing this earlier today :eyes:

**Comment by nhcarrigan** [comment]
@TheBusyBiscuit should be all fixed. Thanks for the help 😁

**Comment by TheBusyBiscuit** [comment]
You're welcome XD

**Pull request merged**